src/index.js

```diff
@@ -166,12 +167,24 @@ export default class RolloverTodosPlugin extends Plugin {
       // Potentially filter todos from yesterday for today
       let todosAdded = 0
       let emptiesToNotAddToTomorrow = 0
-      let todos_today = !removeEmptyTodos ? todos_yesterday : []
+      let todos_today = removeEmptyTodos ? [] : todos_yesterday 
```

Comment:
```suggestion
      let todos_today = !removeEmptyTodos ? todos_yesterday : []
```
Unintentional effect of merge